pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -1166,7 +1166,7 @@ public void validateTopicOperation(TopicName topicName, TopicOperation operation
         }
     }
 
-    protected Void handleCommonRestAsyncException(AsyncResponse asyncResponse, Throwable ex) {
+    protected static Void handleCommonRestAsyncException(AsyncResponse asyncResponse, Throwable ex) {
```

Comment:
I think you can use `resumeAsyncResponseExceptionally`.

Reply:
I think we need to use "handleCommonRestAsyncException" instead of "resumeAsyncResponseExceptionally".
Because using ``handleCommonRestAsyncException`` we don't need to care if it's a ``CompletionException`` exception. And we don't need to "return null" in "exceptionally". I think this change could be clearer.

Thanks for your review, and let me know what you think.

Reply:
You can improve `AdminResource#resumeAsyncResponseExceptionally()` and move to `PulsarWebResource`, so like:
```java
    protected static void resumeAsyncResponseExceptionally(AsyncResponse asyncResponse, Throwable throwable) {
        throwable = FutureUtil.unwrapCompletionException(throwable);
        if (throwable instanceof WebApplicationException) {
            asyncResponse.resume(throwable);
        } else if (throwable instanceof BrokerServiceException.NotAllowedException) {
            asyncResponse.resume(new RestException(Status.CONFLICT, throwable));
        } else {
            asyncResponse.resume(new RestException(throwable));
        }
    }
```

Reply:
fixed, PTAL.